comp/core/workloadfilter/catalog/container.go

```diff
@@ -132,6 +132,12 @@ func LegacyContainerSBOMProgram(config config.Component, logger log.Component) p
 	programName := "LegacyContainerSBOMProgram"
 	var initErrors []error
 
+	if !config.GetBool("sbom.enabled") && !config.GetBool("sbom.container_image.enabled") && !config.GetBool("sbom.container.enabled") {
+		return program.CELProgram{
+			Name: programName,
```

Comment:
If everything is disabled, do we still need to keep the program with this name? Or should we just return an empty CELProgram?